picard/ui/coverartbox.py

```diff
@@ -94,6 +95,19 @@ def __eq__(self, other):
     def show(self):
         self.set_data(self.data, True)
 
+    def decorate_cover(self, pixmap):
+        offx, offy, w, h = (1, 1, 121, 121)
```

Comment:
Just an unrelated note: I wonder about the future of this (hardcoded values), especially since we want to add high dpi compatibility in 2.x releases.